spec/fixtures/crystalball.yml

```diff
@@ -1,5 +1,7 @@
-# Custom path to your maps folder. Default: `tmp/execution_maps`
-map_path: 'execution_map.yml'
+# Custom path to your execution map. Can be file or folder. Default: `tmp/execution_map.yml`
+execution_map_path: 'execution_map.yml'
+# Custom path to your tables map. Default: `tmp/tables_map.yml`
```

Comment:
Please mention a strategy which uses this value

Reply:
Also remember to update here if you remove the default